frontend/src/App.jsx

```diff
@@ -14,4 +16,12 @@ function App() {
   );
 }
 
-export default registerRootComponent(App);
+function reduxApp() {
```

Comment:
please answer to my question below, but if we must stick with a separate component, rename it, it's not a simple function to use camelCase, it's a component, returning JSX. It should be PascalCase with capital first letter.